### PR TITLE
Multiple options for option descriptors

### PR DESCRIPTION
**Pull request by kyle-rosenbluth**
Adds the ability to select multiple options for option descriptors. Also makes sure that CSV input works smoothly, such that you can denote multiple descriptors with semicolons as follows: `desc1;desc2;...`. Also adds placeholder to option descriptors.

### CODE REVIEW

File: app/static/styles/app.css [hunk 1]
```diff
@@ -134,4 +139,5 @@ table.selectable tr:hover {
   font-weight: 300;
   margin-top: 0px;
   text-overflow: ellipsis;
+>>>>>>> origin
```
Comment (kyle-rosenbluth) [inline app/static/styles/app.css]: Fixed

File: migrations/README [hunk 2]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
Comment (kyle-rosenbluth) [inline migrations/README]: Fixed

File: app/single_resource/views.py [hunk 3]
```diff
@@ -50,7 +50,8 @@ def create():
             db.session.commit()
             flash('Resource added', 'form-success')
             return redirect(url_for('single_resource.index'))
-        except IntegrityError:
+        except IntegrityError, e:
```
Comment (kyle-rosenbluth) [inline app/single_resource/views.py]: done

File: app/single_resource/views.py [hunk 4]
```diff
@@ -50,7 +50,8 @@ def create():
...
+            print(e)
```
Comment (kyle-rosenbluth) [inline app/single_resource/views.py]: done

File: app/templates/single_resource/resource_form.html [hunk 5]
```diff
@@ -11,3 +11,10 @@
 
 
 {{ f.render_form(form) }}
+
+<script>
+$(document).ready(function() {
+    console.log($(".ui.dropdown.text"));
```
Comment (kyle-rosenbluth) [inline app/templates/single_resource/resource_form.html]: done

File: app/models/resource.py [hunk 6]
```diff
@@ -7,11 +7,12 @@ class OptionAssociation(db.Model):
     value of the option.
     """
     __tablename__ = 'option_associations'
+    # id = db.Column(db.Integer, primary_key=True)
```
Comment (kyle-rosenbluth) [inline app/models/resource.py]: done

File: app/models/resource.py [hunk 7]
```diff
@@ -7,11 +7,12 @@ class OptionAssociation(db.Model):
     value of the option.
     """
     __tablename__ = 'option_associations'
+
```
Comment (anniemeng) [inline app/models/resource.py]: remove space

**Comment by yoninachmany** [comment]
this would be helpful to have for asylum-connect, so if this could be merged in soon, we'd appreciate it 💯

**Pull request merged**